src/Mod/Sketcher/Gui/DrawSketchController.h

```diff
@@ -372,25 +372,8 @@ class DrawSketchController
             setFocusToOnViewParameter(nextindex);
         }
 
-        /* That is not supported with on-view parameters.

```

Comment:
This looks like something kept because it's nice to have but not possible because of the limitations of OVP. @PaddleStroke any comments?